code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -136,7 +136,7 @@
 	gain_text = "Corrupt their flesh, make them bleed."
 	desc = "Curse someone for 5 minutes of inability to walk. Using a knife, pool of blood, left leg, right leg, a hatchet and an item that the victim touched  with their bare hands. "
```

Comment:
Curse someone for 5 minutes with the inability to walk. Using a left leg, a right leg, a hatchet and an item that the victim touched with their bare hands. 

should be the new desc for this